godaddy/README.md

```diff
@@ -1,13 +1,45 @@
+# GoDaddy
+
 ## Overview
-The GoDaddy integration seamlessly collects metrics from SSL certificates and domains, directing them into Datadog for analysis. This integration provides insights including total certificates, issued certificates, expired certificates, revoked certificates, count of domains associated with each certificate. It features statistics for certificates that are about to expire along with many additional metrics, all accessible through out-of-the-box dashboards and monitors.
+GoDaddy is a prominent web hosting and domain registration company that provides a variety of services to help individuals and businesses establish their online presence. A notable aspect of GoDaddy product lineup is their SSL certificate services. GoDaddy offers a range of SSL certificate options to suit different needs, including standard SSL for securing one site, Wildcard SSL for securing multiple subdomains, and more advanced solutions for ecommerce sites requiring the highest level of security.
+
+The GoDaddy integration seamlessly collects metrics from SSL certificates and their domains, directing them into Datadog for analysis. This integration provides insights including total certificates, issued certificates, expired certificates, revoked certificates, count of domains associated with each certificate. It features statistics for certificates that are about to expire along with many additional metrics, all accessible through out-of-the-box dashboards and monitors.
 
 ## Setup
 
-### Configuration
-!!! Add list of steps to set up this integration !!!
+### Get API Credentials from GoDaddy
+
+#### GoDaddy API KEY and API Secret
+
+- Go to [GoDaddy Developer Portal][1]
+- Sign in with your GoDaddy account.
+- Select "API Keys".
+- Choose "Create New API Key".
+- Provide a name for your API.
+- Select "Production" under Environment.
+- Click "Next".
+- Your API Key is now created.
+- Copy these credentials for the subsequent configuration steps.
+- Ensure these credentials are stored securely and not exposed in public repositories or insecure locations.
+- After preserving your API Key and Secret in a separate document, click on "Got It".
+
+#### GoDaddy Customer Number (ShopperID)
+
+- Go to your GoDaddy [Login & PIN page][2]. You might be prompted to sign in.
+- Under Login Info, find your Customer number(ShopperId).
+
+### GoDaddy Datadog Integration Configuration
+
+Configure the Datadog endpoint to forward GoDaddy metrics to Datadog.
+
+1. Navigate to `GoDaddy`.
+2. Add your GoDaddy credentials.
 
-### Validation
-!!! Add steps to validate integration is functioning as expected !!!
```

Comment:
I'm not sure the table below adds much value? The "parameter" and "description" are almost identical on each line, so maybe we can get rid of this altogether?

Reply:
removed the table and directly added the parameter.